### ISSUE TITLE
[Feature Request] PoE (Power over Ethernet) variant of the ZiGate-Ethernet hardware?

### ISSUE THREAD

**Issue by Hedda** [fairecasoimeme/ZiGate-Ethernet#1]
Please consider making a PoE (Power over Ethernet) version of the ZiGate-Ethernet hardware so it can be powered over Ethernet.

Reason for wanting PoE support is that it is great when you want to place the gateway in a place where there is no other power.

For ideas, could look at Olimex ESP32-POE-ISO and/or LILYGO TTGO T-Internet-POE which looks to be open source hardware: 

https://www.olimex.com/Products/IoT/ESP32/ESP32-POE-ISO/open-source-hardware

https://github.com/OLIMEX/ESP32-POE-ISO

http://www.lilygo.cn/prod_view.aspx?TypeId=50033&Id=1307

https://github.com/Xinyuan-LilyGO/LilyGO-T-ETH-POE

As an example of someone implementing Olimex ESP32-POE-ISO for this purpose see Tube's PoE Zigbee Gateway by tube0013

https://www.tubeszb.com/product/cc2652_poe_coordinator/15

https://github.com/tube0013/tube_gateways

https://twitter.com/TubesZb/status/1394478007730020352

https://twitter.com/TubesZb/status/1394845581823074308

PS: Other implementations for

**Comment by fairecasoimeme** [comment]
Hi,
POE is really good option but for me It have to be an option, I work about this feature and there will be modification on the future version.

Thanks

**Comment by Hedda** [comment]
> POE is really good option but for me It have to be an option

I agree that it would be best if PoE could be enabled via an optional addon board or via an automatic or manual switch.

I guess that most people will probably want USB-power so that option should always be available unless you would want to make and sell two separate products.

If you look at ZOE and ZOE2 (Zoe 2) PoE and Zigbee shield for RPi by Electrolama they have made it as an addon board

https://electrolama.com/projects/zoe2/

https://electrolama.com/projects/zoe/

https://github.com/electrolama/zoe2

https://github.com/electrolama/zoe

The downside with an addon board is increased complexity and cost for those people who do want to use PoE
